clothing_rental/accounts/views.py

```diff
@@ -1,7 +1,29 @@
-from django.shortcuts import render
-from django.http import HttpResponse
+from django.shortcuts import render, redirect
+from django.contrib.auth import get_user_model, login
+from .forms import UserRegistrationForm
```

Comment:
poprawmy grupowanie importów wg pep8

Reply:
from accounts.forms import UserRegistrationForm?

Reply:
taką formę bym wolał, aczkolwiek przede wszystkim chodzi o poprawne grupowanie importów

https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement